src/main/java/uk/gov/hmcts/reform/bsp/common/service/transformation/BulkScanFormTransformer.java

```diff
@@ -29,6 +31,10 @@
         return caseData;
     }
 
+    protected Map<String, Object> transformAdditionalDataFromExceptionRecord(ExceptionRecord exceptionRecord) {
+        return Collections.emptyMap();
```

Comment:
is this unfinished?

Reply:
No. It's finished.
The default behaviour for this method (when not overridden) is to add no new fields to the caseData (empty map).